controllers/scoring.py

```diff
@@ -80,8 +80,10 @@ def initialize(self, initial_call: bool, state_tm: float) -> None:
             self.arm.homing = False
             self.arm.go_to_setpoint(Setpoints.STOW)
 
-        if (self.arm.get_angle() < 0.5 and not self.arm.homing) or (
-            wpilib.DriverStation.isAutonomous() and state_tm > 0.5
+        if (
+            (self.arm.get_angle() < 0.5 and not self.arm.homing)
+            or (wpilib.DriverStation.isAutonomous() and state_tm > 0.5)
+            or wpilib.RobotBase.isSimulation()
```

Comment:
```suggestion
        if (
            (self.arm.get_angle() < 0.5 and not self.arm.homing)
            or (wpilib.DriverStation.isAutonomous() and state_tm > 0.5)
```

Not required anymore since I merged #77